package.json

```diff
@@ -202,6 +205,7 @@
             "node_modules/@mongodb-js/compass-instance",
             "node_modules/@mongodb-js/compass-serverstats",
             "node_modules/@mongodb-js/compass-server-version",
+            "node_modules/@mongodb-js/compass-shell",
```

Comment:
does this mean the next version of compass will have the shell?

Reply:
Yes. Maybe we leave this PR branch up and when we're ready to merge we do it. Not sure if we want to have this in the next release or if we want to do a smaller release before.